src/Parser/RestorationShaman/Modules/ShamanCore/ShamanAbilityTracker.js

```diff
@@ -44,6 +48,22 @@ class ShamanAbilityTracker extends AbilityTracker {
     }
 
   }
+
+  suggestions(when) {
+    const healingRain = this.getAbility(SPELLS.HEALING_RAIN_CAST.id);
+    const has4PT20 = this.owner.selectedCombatant.hasBuff(SPELLS.RESTORATION_SHAMAN_T20_4SET_BONUS_BUFF.id);
+    const unbuffedHealingRainsPercentage = has4PT20 && ((healingRain.casts - healingRain.withT20Buff) / healingRain.casts);
+    if (has4PT20) {
```

Comment:
This should be moved to a 4 piece T20 module and not be placed here as it would be hard to find.